### PR TITLE
First draft of jobs page with two open positions

### PR DESCRIPTION
**Pull request by niallwingham**
This adds a new jobs page to our website, with details for the Software Developer and Project/Business Manager we are trying to hire.  Please review all copy.

There are a few open questions I have to finish these pages:

1. How long do we estimate the work sample takes?  (And am I correct that we don't have a time limit on it?)
2. Do we know the _average_ vacation taken by our team?  Common question is okay you have a 3-week minimum but is that in practice a maximum as well?  If we don't have hard data a rough estimate will do.
3. Can we add details for retirement contribution and health insurance benefits?
4. Can we make an explicit parental leave policy?
5. Are there any additional responsibilities you would like to add to the Project/Business Manager?

Instead of applying by email, I'm asking candidates to fill out this form: https://www.cognitoforms.com/ApsisLabs1/SoftwareDeveloperApplicationForm.  You should review the form as part of this pull request.  Once we agree on it I'll make a similar form for the project/business manager.  It's in trial mode right now and will cost $25/month to run for real.

At the end of the form is a diversity questionnaire we can use for tracking which job boards are sending us diverse & qualified candidates, as well as to identify possible bias in our recruitment/interview process.  I am pretty sure this is legal; see my comment below for support and discussion on this point.

For tracking, I'll be inputing completed forms into a hiring project I've set up on Jira with a custom workflow to match our interview process.

### CODE REVIEW

**Comment by niallwingham** [comment]
Here is the basis for my thinking the diversity questionnaire is legal:

> **Can an employer ask about an applicant's race on an application form?**  Employers may legitimately need information about their employees or applicants race for affirmative action purposes and/or to track applicant flow.

https://www.eeoc.gov/policy/docs/qanda_race_color.html

> Additionally, proactively keeping track of applicants and their race and gender designations can help contribute to a more diverse work environment. By tracking demographic data, HR can periodically review its recruiting strategy and hiring decisions and adjust any contributing factors in order to create a more diverse applicant pool and in turn a more diverse workforce.

https://www.shrm.org/resourcesandtools/tools-and-samples/hr-qa/pages/shouldemployerscollectdemodataonappsandemployees.aspx

As well as these, which are not legal advice, but are relevant precedents and calls to action within the tech community:

> We began collecting this information on a voluntary basis on May 18. All the applicant information comes from a Wufoo form that those who apply for a role at Buffer are invited (though not required) to share with us after their application is submitted.

https://open.buffer.com/diversity-dashboard/

> Track the gender of your applicants, not just the hires.

https://www.hiremorewomenintech.com/

**Comment by niallwingham** [comment]
I think it's important to go beyond the word-of-mouth / social media recruiting we've done so far.  The EEOC specifically warns against that type of recruitment as well:

> ### How can employers avoid racial discrimination when recruiting?
>  - **Word-of-mouth employee referrals** Word-of-mouth recruitment is the practice of using current employees to spread information concerning job vacancies to their family, friends, and acquaintances. Unless the workforce is racially and ethnically diverse, exclusive reliance on word-of-mouth should be avoided because it is likely to create a barrier to equal employment opportunity for racial or ethnic groups that are not already represented in the employer's workforce.
>  - **Homogeneous recruitment sources** - Employers should attempt to recruit from racially diverse sources in order to obtain a racially diverse applicant pool. For example, if the employer's primary recruitment source is a college that has few African American students, the employer should adopt other recruitment strategies, such as also recruiting at predominantly African American colleges, to ensure that its applicant pool reflects the diversity of the qualified labor force.

https://www.eeoc.gov/policy/docs/qanda_race_color.html

Here are the job boards I want to post to, at least to start with:

- Hacker News who's hiring (free)
- Twitter, Facebook, Reddit, etc. (free)
- We Work Remotely (https://weworkremotely.com/jobs/new) $300
- Authentic Jobs (https://www.authenticjobs.com/post) $300
- Hire Tech Ladies (https://www.hiretechladies.com/partners) inquire
- Power to Fly (https://powertofly.com/employers/) inquire
- Fairy God Boss (https://fairygodboss.com/employers) inquire

The boards listed "inquire" have an application process to become an employer, to make sure we fit with their ethos.  I'm not sure what the price to post will be after completing the application, but I'd guess they're each a few hundred dollars as well.

Here are some job boards that I considered but do not want to start with:

- Stack Overflow (dev only) $3875 **prefer to try cheaper options first**
- GitHub (https://jobs.github.com/post#create) $450 **prefer to try cheaper options first**
- Slack (https://slackatwork.com/post-a-job/) **small board and requires paid slack account**
- Dice (http://www.dice.com/products/webstore) $395 **don't think it offers an audience not already captured above, no filter for remote jobs**
- FlexJobs (https://www.flexjobs.com/) inquire **not focused on software**
- Doxa (http://doxascore.com) inquire **seems to require at least 60 employees**
- Advancing Women (http://careers.advancingwomen.com/employer/post) $250 **not focused on software**
- Women in Technology International (http://jobs.witi4hire.com/r/jobs/post/index.cfm?site_id=273) $320 **not focused on software**

Let me know if you have any other job boards to recommend!

File: jobs/software-developer.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        <p>Some of our current frameworks are Ruby on Rails, React + Redux, and Xamarin.  It’s okay if you’re not an expert in those&mdash;we can train you.  We are especially interested in hiring developers who bring expertise in a different framework we may use in the future, and who are interested in learning new technologies, idioms, and patterns.</p>
+        <p>Finally, because (a) much of our work is client-facing and (b) we want to work efficiently as a remote team without resorting to heavy handed management, we look for candidates who are self-starters and have good communication and collaboration skills.</p>
+
+        <h3 id="process">Interview Process</h3>
+        <p>Our standard process is two phone interviews and a final onsite interview in Seattle, WA.  You may <a href="#customize">customize the process</a> if you don’t do well in phone interviews or are unable to travel.</p>
```
Comment (HenryKeiter) [inline jobs/software-developer.html]: Someone please test that this scroll ID still works. My local env is broken and I'm not wasting time with that right now.

**Comment by wkirby** [comment]
Jekyll just generates static HTML. We'd need some sort of backend to
process the form, and GitHub pages isn't interesting in hosting things like
that.

**Comment by HenryKeiter** [comment]
Should we look into a cheaper hosting solution for something like `jobs.apsis.io`, to give us more control over our application form? That way we could (presumably) hook it up to whatever tracking services we want to use.

**Comment by niallwingham** [comment]
Thanks for the commits Henry!

For the form, I thought about building it myself but figured this was good enough for now and a lot cheaper (given an opportunity cost of $200/hr for time spent on this instead of client work).

We can embed the form on our website if we want--though the edit submission link will always take them to cognito--and once we're out of trial it doesn't have cognito branding in either mode.

(I also invited you to the cognito account in case you want to poke around there.)

**Comment by niallwingham** [comment]
Collected feedback from Noah offline and resolved the remaining copy issues.

We're going to start with free job boards + we work remotely, and do additional postings depending on results there.

Diversity questionnaire is good.

**Pull request merged**